docker/Dockerfile

```diff
@@ -11,6 +11,7 @@ ARG SHA1="[no-sha]"
 ARG TAG="[no-tag]"
 ARG GOARCH
 
+RUN apk add --no-cache bash
```

Comment:
What's this about?

Reply:
While enabling sentinel for redis, it expects the exporter to have bash with it.
So, to make it compatible with redis bitnami sentinel, raised this.